### DISCUSSION TITLE
Minor post-op supported?

### DISCUSSION THREAD

**Discussion by zhongyilin926** [OpenMathLib/OpenBLAS#4759] [Q&A]
Hi, I was wondering if OpenBLAS supports (the fusion of) minor post-op e.g. bias-add right after GEMM? An example would be something like what `torch.addmm` or `torch.baddbmm` do. Thank you!

**Comment by martin-frbg** [comment]
No, this is currently not supported - the current develop branch (and upcoming 0.3.28 release) has the gemm_batch operations, but not this (apparently Torch-specific) post-op extension.
